azure-data-lake/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -35,40 +45,33 @@ These are some common use cases for Azure Data Lake Connector:
 To create your Mule app, see:
 
 * xref:azure-data-lake-connector-studio.adoc[Studio Configuration] 
-* xref:azure-data-lake-connector-config-topics.adoc[Additional Configuration Information]
-to help extend your understanding of Azure Data Lake Storage Connector 
 * xref:azure-data-lake-connector-examples.adoc[Examples], with use case information for the connector
 +
 * Advanced user
 +
 * Read the following: 
 
 * xref:azure-data-lake-connector-xml-maven.adoc[XML and Maven Support]
-* xref:azure-data-lake-connector-config-topics.adoc[Additional Configuration Information]
 * xref:azure-data-lake-connector-examples.adoc[Examples], with use case information for the connector
 
-// The following line is only needed for connectors with upgrade information
-
-For a major or minor release, see
-xref:azure-data-lake-connector-upgrade-migrate.adoc[Upgrade and Migrate].
-
 == Connection Types
 
 Azure Data Lake Storage Connector connections use the following authentication types:
 
 * Azure Active Directory OAuth 2.0 Connection Provider +
 Delegates user authentication to the service hosting the user account
-* Share Access Signature
-* Shared Key Connection
-
-For information about configuring authentication types, see xref:azure-data-lake-studio.adoc[Use Studio to Configure Azure Data Lake Storage Connector].
+* Shared Access Signature +
+Information contained in the SAS token is used to authorize the client request
```

Comment:
Reword to make a fragment like its peer...or reword the peer to be a sentence. Parallelism.
If you choose sentences, don't forget the periods.